### PR TITLE
Made default redirection to library page

### PR DESCRIPTION
**Pull request by Bahugunajii**
### Description
- Separated server component from client component
- Made default redirection to library page (temp.)

### Checklist
- [x] Self testing

### CODE REVIEW

File: api/afdb/library.ts [hunk 1]
```diff
@@ -1,8 +1,10 @@
+"use server"
```
Comment (pritamps) [inline api/afdb/library.ts:1]: Is this needed? `server` is default, right?

File: api/reporting/reports.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+"use server"
```
Comment (pritamps) [inline api/reporting/reports.ts:1]: Same question

File: api/afdb/library.ts (hunk 1 above)
Comment (Bahugunajii) [reply api/afdb/library.ts:1]: yes but I'm using server actions here

**Review by sudheshna-donthineni — APPROVED** [review]

File: api/reporting/reports.ts (hunk 2 above)
Comment (Bahugunajii) [reply api/reporting/reports.ts:1]: replied above

**Pull request merged**